share/man/man4/acpi_wmi.4

```diff
@@ -56,7 +56,7 @@ WMI status device.
 .El
 .Sh SYSCTLS
 The following sysctl node is currently implemented:
-.Bl -tag 
+.Bl -tag
 .It Va dev.acpi_wmi.%d.bmof
 Managed Object Format (MOF) blob.
 You can obtain human readable output by bmf2mof in bmfdec tool.
```

Comment:
Is there any way, while we're here, We can get that url converted to a Pq Lk? That makes it actually followable in most software. It's still within the scope of "minor cleanup (markup)", and I'm not (a SME or) aware of anything else that really needs done to this page.

Reply:
Good catch!  (although I still don't think it's in scope of this particular PR)

I think that's a good candidate for a script, though.
```
$ grep "http://" -- $(cat man-pages.txt) | grep -v "\.\\\\\"" | grep -v "Lk " | wc -l
     169
$ grep "https://" -- $(cat man-pages.txt) | grep -v "\.\\\\\"" | grep -v "Lk " | wc -l
     922
```

Amusingly, there's quite a notable amount where people used a `.Pa` macro in front of it -- 55 for `http:`, and 24 for `https:`.

Reply:
After reviewing the doc like I'm supposed to do while we're proposing changing it, it turns out that repo is packaged, so I've moved it out of the sysctl and into the example where it's used and linked it here https://github.com/freebsd/freebsd-src/pull/1477.